packages/site/src/components/Account.tsx

```diff
@@ -68,16 +76,18 @@ export const Account = ({
               ))}
             </ul>
           </AccountRow> */}
-          <AccountRow alignItems="flex-end">
-            <MethodButton
-              width="30%"
-              margin="8px 0px 8px 8px"
-              onClick={async (): Promise<void> => {
-                await handleDelete(account.id);
-              }}
-              label="Delete"
-            />
-          </AccountRow>
+          {currentAccount.toLowerCase() === account.address.toLowerCase() ? <></> :
```

Comment:
minor: I was thinking about the case where - there's only one account and then the user cannot delete the account. should we keep the option for users to delete the account through the page?

If the account is not deleted through the page - the snap local state would still have it and could cause issues

Reply:
hmm, I agreed, so better let me remove this condition completely and keep delete button visible all the time!